src/client/index.ts

```diff
@@ -251,7 +258,7 @@ export class Client {
 
 		if (this.config.type === "active") {
 
-			this.issuersLoaded = !this.tokenStore.hasUnloadedIssuers();
+			this.issuersLoaded = false;
```

Comment:
I'll review this in a bit more detail - to ensure that at this point the setting should be false.